CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [1.25.1] - 2024-12-20
+
 ### Added
 
 - Added validation for required flags for the `rsconnect system caches delete` command.
```

Comment:
it would be nice to pull in the latest commit to `main` for this release since it is a fix for a user-facing issue. what do you think?

https://github.com/posit-dev/rsconnect-python/commit/9478983e4ca5aed71177467019736a9671590123

Reply:
done!